x-pack/plugins/logstash/server/models/cluster/cluster.ts

```diff
@@ -23,8 +24,8 @@ export class Cluster {
   }
 
   // generate Pipeline object from elasticsearch response
-  static fromUpstreamJSON(upstreamCluster) {
-    const uuid = get(upstreamCluster, 'cluster_uuid');
+  static fromUpstreamJSON(upstreamCluster: Record<string, string>) {
+    const uuid = get<string>(upstreamCluster, 'cluster_uuid');
```

Comment:
Can probably be replace with `upstreamCluster.cluster_uuid` ?

Reply:
Frankly speaking, I don't know whether `upstreamCluster` always presents in response. This type is my guess, so I wouldn't take a risk to change this code https://github.com/elastic/kibana/blob/877fa0fda0eb2de896df30dc676e98457fbd47b5/x-pack/plugins/logstash/server/routes/cluster/load.ts#L22